apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -392,27 +429,51 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
 
         if (selectedDialect.localDialect) {
             if (claimMappingOn) {
+                let usernameAdded: boolean = false;
                 const claimMappingOption: DropdownOptionsInterface[] = [];
+                const claimMappingList = [...claimMapping];
                 claimMapping.map((element: ExtendedClaimMappingInterface) => {
-                    let option: DropdownOptionsInterface;
-                    if (!isEmpty(element.applicationClaim)) {
-                        option = {
-                            key: element?.localClaim?.id,
+                    if (!element || !element.localClaim) {
+                        return;
+                    }
+                    const option: DropdownOptionsInterface = {
+                        key: element?.localClaim.uri,
+                        text: (
+                            <SubjectAttributeListItem
+                                key={ element.localClaim.id }
+                                displayName={ element.applicationClaim ?
```

Comment:
Add undefined check

Reply:
Undefined check is added in the beginning of the map function